index.html

```diff
@@ -0,0 +1,733 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link href="https://fonts.googleapis.com/css2?family=Maven+Pro:wght@600&display=swap" rel="stylesheet">
+    <link href="https://fonts.googleapis.com/css2?family=Heebo:wght@300&display=swap" rel="stylesheet">
+    <script src="https://kit.fontawesome.com/9a2f12287f.js" crossorigin="anonymous"></script>
+    <link rel="stylesheet" href="style.css">
+    <link rel="stylesheet" href="responsive.css">
+    <title>TNW</title>
+</head>
+
+<body>
+    <header>
+        <div class="medium-nav">
+        <nav class="nav1">
+            <ul>
+                <li>News</li>
+                <li>Events <i class="fas fa-angle-down"></i> </li>
+                <li>Business <i class="fas fa-angle-down"></i> </li>
+                <li>AMAs</li>
+                <li>Spaces</li>
+                <li>Terms & Conditions</li>
+            </ul>
+            <ul>
+                <li>About</li>
+                <li>Advertise</li>
+                <li>Jobs</li>
+                <li>Contact</li>
+            </ul>
+        </nav>
+        <nav class="nav2">
+            <ul>
+                <li>LATEST</li>
+                <li>HARD FORK</li>
+                <li>PLUGGED</li>
+                <li>README</li>
+                <li>GROWTH QUARTERS</li>
+                <li>NEURAL</li>
+            </ul>
+            <div class="nav-search">
+                <i class="fas fa-search"></i>
+            </div>
+        </nav>
+        <div class="mobile-nav2">
+            <img class="tnw-red" src="img/tnw.svg" alt="tnw logo">
+        </div>
+        </div>
+        <nav class="mobile-nav">
+           
+            <div class="mobile-img">
+                <img class="tnw-red" src="img/tnw.svg" alt="tnw logo">
+            </div>
+            <div class="mobile-nav1">
+                <div class="mobile-nav-search">
+                    <i class="fas fa-search gray"></i>
+                </div>
+                <div class="mobile-nav-more">
+                    <i class="fas fa-bars orange"></i>
+                </div>
+            </div>
```

Comment:
- [ ] On **mobile** Please check that sometimes your navbar gets behind your images, to fix this, you can add the `index-z: (number);` rule in your CSS
![image](https://user-images.githubusercontent.com/53256139/88814452-bbcc4680-d17f-11ea-8980-3fc2a99b9cba.png)